### ISSUE TITLE
Installing monoloco using pip

### ISSUE THREAD

**Issue by mvssanjay007** [vita-epfl/monoloco#34]
When i try to install monoloco using command : pip3 install monoloco, it is trying to install torch but couldn't. Is there any mistake on my side in the process of installation? I am beginner in the field of deep learning and AI.  
![Screenshot from 2020-08-20 01-40-04](https://user-images.githubusercontent.com/48452854/90684738-56530d80-e286-11ea-86eb-b8fcac7e11d0.png)
![Screenshot from 2020-08-20 01-40-14](https://user-images.githubusercontent.com/48452854/90684742-581cd100-e286-11ea-9ea1-bfd97e3e08c0.png)
![Screenshot from 2020-08-20 01-40-19](https://user-images.githubusercontent.com/48452854/90684745-58b56780-e286-11ea-91ae-094fb1e13a8d.png)
![Screenshot from 2020-08-20 01-40-24](https://user-images.githubusercontent.com/48452854/90684747-58b56780-e286-11ea-8053-b4db5973a661.png)
![Screenshot from 2020-08-20 01-40-26](https://user-images.githubusercontent.com/48452854/90684749-594dfe00-e286-11ea-84e5-8607a4d986d6.png)
![Screenshot from 2020-08-20 01-40-29](https://user-images.githubusercontent.com/48452854/90684753-59e69480-e286-11ea-87cb-2651d5d80a39.png)

**Comment by bertoni9** [comment]
Hello,
It seems you have already installed torch 0.1.2, which is incompatible with pifpaf 0.9. I suggest you to start with a fresh virtualenv.
Best,
Lorenzo

**Comment by mvssanjay007** [comment]
When I tried in fresh virtual env, I got the same issue. So i downloaded the tar.gz file from pip webiste for monoloco and found the setup.py . In setup.py file I found the requirement version for torch and torchvision. They were not compatible as you mentioned above. openpifpaf 0.9 is required for monoloco but openpifpaf 0.9 require torch>=1.1.0 and monoloco is intalling torch <1.1.0
![Screenshot from 2020-08-20 16-43-00](https://user-images.githubusercontent.com/48452854/90769954-b097b100-e30e-11ea-8687-7d46abdfc00b.png)
